pytorch_blade/tests/disc/testing_base.py

```diff
@@ -211,3 +211,21 @@ def setUp(self):
         super().setUp()
         if self.device != torch.device('cuda'):
             self.skipTest("Quantization pdl test case only supports gpu platform")
+
+    def _test_e2e(
+            self, model, inp, pdll_files=None,
+            pdll_dirs=None, enable_int8=False
+    ):
+        origin_output = model(inp)
+        cfg = Config.get_current_context_or_new()
+        cfg.optimization_pipeline = mlir.backend_name()
+        cfg.enable_int8 = enable_int8
+        env_var = {}
+        if pdll_files is not None:
+            env_var["DISC_TORCH_PDL_FILES"] = pdll_files
+        if pdll_dirs is not None:
+            env_var["DISC_TORCH_PDLL_INCLUDE_DIRS"] = pdll_dirs
+        with set_env(**env_var), cfg:
+            opt_model = optimize(model, True, inp)
+        now_output = opt_model(inp)
+        self.assertTrue(torch.abs(now_output-origin_output).max() <= 0.3)
```

Comment:
1. why there is a diff？
2. do not hard code the threshold
3. use `torch.allclose` instead